api/src/main/java/io/strimzi/api/kafka/model/kafka/listener/KafkaListenerAuthenticationOAuth.java

```diff
@@ -285,6 +287,16 @@ public void setIntrospectionEndpointUri(String introspectionEndpointUri) {
         this.introspectionEndpointUri = introspectionEndpointUri;
     }
 
+    @Description("Path to the file on the local filesystem that contains a bearer token to be used instead of client_id and secret when authenticating to authorization server")
```

Comment:
Same as earlier -> is this required for `type: serviceaccount-oauth`? If not, we should maybe add the default. Otherwise say that it has to be used in the description.